packages/graph/src/Graph.ts

```diff
@@ -285,6 +285,12 @@ export class Graph extends SVGZoomWidget {
     enter(domNode, element) {
         super.enter(domNode, element);
 
+        this._zoomGrab.on("click.clear", () => {
+            if (this.selectionClearOnBackgroundClick()) {
```

Comment:
Clicking on the background seems like a clear intent to deselect. I don't think it needs to be an option to behave otherwise... unless that background click serves some other functionality?

Reply:
Ah okay that makes sense. Should I open an issue to deselect on background click for all widgets that support selection?